docs/human-interface-guidelines.md

```diff
@@ -618,36 +606,61 @@ Most search fields should use the format "Search OBJECTS" where OBJECTS is somet
 
 If the search field interacts with a search service, the hint text should be the name of that service such as "Google" or "Yahoo!"
 
-## Checkboxes & Switches {#checkboxes-switches}
+<span id="checkboxes-switches"/>
+## Selection Controls {#selection-controls}
+
+Selection controls present a way for users to select or enable options. There are several types of selection controls available in elementary OS:
 
+* **Checkboxes** present a way for users to select multiple items from a set.
+* **Comboboxes**, **Linked buttons** , and **Radio buttons** present a way for users to select a single option from a set.
+* **Switches** present a way for users to toggle certain features or behaviors "on" or "off".
+
+<span id="checkboxes-usage"/><span id="checkboxes-labeling"/>
 ### Checkboxes {#checkboxes}
 
-Checkboxes present a way for users to select items from a list.
+Use checkboxes when users are making a selection of items. If you have a single option, avoid using a checkbox and use a switch instead.
 
-#### Usage {#checkboxes-usage}
+Make sure that users can toggle the state of the checkbox by clicking on the label associated with the checkbox. 
 
-Use checkboxes when users are making a selection of items.
-Make sure that a user can toggle the state of the checkbox by clicking on the label associated with the checkbox.
+Labels associated with checkboxes should usually be nouns or nounal phrases.
 
-#### Labeling {#checkboxes-labeling}
+### Comboboxes {#comboboxes}
 
-Labels associated with Checkboxes should usually be nouns or nounal phrases.
+Use a combobox (also called a dropdown) when:
 
-### Switches {#switches}
+* Users are selecting only a single item from a set and
+* The set is too long to show all available options at once
 
-![](/images/docs/human-interface-guidelines/checkboxes-switches/switches.png)
+<span id="linked-buttons-usage"/><span id="linked-buttons-labeling"/>
+### Linked Buttons {#linked-buttons}
 
-Switches present a way for users to toggle certain features or behaviors "on" or "off".
+![](/images/docs/human-interface-guidelines/buttons/radio.png)
 
-#### Usage {#switches-usage}
+Use linked buttons (also called a mode button) when:
```

Comment:
again with the One True Widget Name here. What do we want the canonical name to be, and shouldn't we just use that?